vaxrank/epitope_prediction.py

```diff
@@ -25,68 +25,47 @@
 
 logger = logging.getLogger(__name__)
 
-EpitopePredictionBase = namedtuple(
-    "EpitopePrediction", [
-        "allele",
-        "peptide_sequence",
-        "length",
-        "ic50",
-        "percentile_rank",
-        "prediction_method_name",
-        "overlaps_mutation",
-        "source_sequence",
-        "offset",
-        "occurs_in_reference",
-    ])
-
-class EpitopePrediction(EpitopePredictionBase):
-
-    @classmethod
-    def from_mhctools_binding_prediction(
-            cls,
-            binding_prediction,
-            overlaps_mutation,
-            occurs_in_reference):
-        return cls(
-            allele=binding_prediction.allele,
-            peptide_sequence=binding_prediction.peptide,
-            length=len(binding_prediction.peptide),
-            ic50=binding_prediction.value,
-            percentile_rank=binding_prediction.percentile_rank,
-            prediction_method_name=binding_prediction.prediction_method_name,
-            overlaps_mutation=overlaps_mutation,
-            source_sequence=binding_prediction.source_sequence,
-            offset=binding_prediction.offset,
-            occurs_in_reference=occurs_in_reference)
-
-    def logistic_score(
-            self,
-            midpoint=350.0,
-            width=150.0,
-            ic50_cutoff=2000.0):
-        """
-        Map from IC50 values to score where 1.0 = strong binder, 0.0 = weak binder
-        Default midpoint and width for logistic determined by max likelihood fit
-        for data from Alessandro Sette's 1994 paper:
-
-           "The relationship between class I binding affinity
-            and immunogenicity of potential cytotoxic T cell epitopes.
-
-        TODO: Use a large dataset to find MHC binding range predicted to #
-        correlate with immunogenicity
-        """
-        if self.ic50 >= ic50_cutoff:
-            return 0.0
-
-        rescaled = (float(self.ic50) - midpoint) / width
-        # simplification of 1.0 - logistic(x) = logistic(-x)
-        logistic = 1.0 / (1.0 + np.exp(rescaled))
-
-        # since we're scoring IC50 values, let's normalize the output
-        # so IC50 near 0.0 always returns a score of 1.0
-        normalizer = 1.0 / (1.0 + np.exp(-midpoint / width))
-
-        return logistic / normalizer
+EpitopePrediction = namedtuple("EpitopePrediction", [
+    "allele",
+    "peptide_sequence",
+    "length",
+    "ic50",
+    "percentile_rank",
+    "prediction_method_name",
+    "overlaps_mutation",
+    "source_sequence",
+    "offset",
+    "occurs_in_reference",
+])
+
+def logistic_epitope_score(
```

Comment:
why the change to make this a method outside the EpitopePrediction object? Looks like an unrelated refactor - we should stick to one (logical) change per PR, with the exception of adding comments/fixing whitespace/etc.